### DISCUSSION TITLE
Update Build Instructions?

### DISCUSSION THREAD

**Discussion by thomasgu112** [Inori/GPCS4#111] [Suggestions]
I can't figure out how to build the latest version. I tried using VS 2019 (I think that is what is intended), but I get strange errors coming from the clang headers. More concretely, when building pthreads4w I get this:
![errors1](https://user-images.githubusercontent.com/60446773/161087193-6b03078c-9445-4190-9afb-ffd7c851b340.png)
And when building TinyDBR I get this:
![errors2](https://user-images.githubusercontent.com/60446773/161087369-07d4b387-e4e1-48dc-bbe7-76b5c3062ba1.png)
Building tag v0.11 using VS 2017 according to instructions works fine.

**Comment by Inori** [comment]
OK，I'll update build document soon.

**Comment by Inori** [comment]
I'v done it, although there are some ugly steps need to be fixed.
You can have a try.

**Reply by thomasgu112** [reply]
Yes, with that I can build now. And by the way thank you for your pioneering work on this software, I am excited to see where it goes!
